### PR TITLE
add japanese (ja) locale

### PR DESCRIPTION
**Pull request by kiwamizamurai**
I just added the Japanese locale

### CODE REVIEW

File: Pearcleaner/Resources/Localizable.xcstrings [hunk 1]
```diff
@@ -1358,10 +1538,16 @@
...
+        "ja" : {
+          "stringUnit" : {
+            "state" : "translated",
+            "value" : "クリア"
+          }
         }
       }
     },
-    "Clear text" : {
+        "Clear text" : {
```
Comment (kiwamizamurai) [inline Pearcleaner/Resources/Localizable.xcstrings:1550]:
```suggestion
    "Clear text" : {
```

File: Pearcleaner/Resources/Localizable.xcstrings [hunk 2]
```diff
@@ -9173,10 +10383,16 @@
...
+        "ja" : {
+          "stringUnit" : {
+            "state" : "translated",
+            "value" : "表示"
+          }
         }
       }
     },
-    "View in Finder" : {
+     "View in Finder" : {
```
Comment (kiwamizamurai) [inline Pearcleaner/Resources/Localizable.xcstrings:10395]:
```suggestion
    "View in Finder" : {
```

File: Pearcleaner/Resources/Localizable.xcstrings [hunk 3]
```diff
@@ -4146,14 +4704,20 @@
...
-          }
-        }
-      }
-    },
-    "Locations that will be searched for .app files. Click a non-default path to remove it. Default paths can't be removed." : {
+           "stringUnit" : {
+             "state" : "translated",
+             "value" : "登入時啟動 Pearcleaner"
+           }
+         },
```
Comment (kiwamizamurai) [inline Pearcleaner/Resources/Localizable.xcstrings:4711]:
```suggestion
        },
```

File: Pearcleaner/Resources/Localizable.xcstrings [hunk 4]
```diff
@@ -4139,21 +4697,27 @@
...
+             "state" : "translated",
+             "value" : "登入時啟動 Pearcleaner"
+           }
+        },
+        "ja" : {
+           "stringUnit" : {
+             "state" : "translated",
+             "value" : "ログイン時にPearcleanerを起動"
+           }
+         }
```
Comment (kiwamizamurai) [inline Pearcleaner/Resources/Localizable.xcstrings:4717]:
```suggestion
        }
```

File: Pearcleaner/Resources/Localizable.xcstrings [hunk 5]
```diff
@@ -4146,14 +4704,20 @@
...
+             "state" : "translated",
+             "value" : "ログイン時にPearcleanerを起動"
+           }
         }
-      }
-    },
-    "Locations that will be searched for .app files. Click a non-default path to remove it. Default paths can't be removed." : {
+       }
+     },
+     "Locations that will be searched for .app files. Click a non-default path to remove it. Default paths can't be removed." : {
```
Comment (kiwamizamurai) [inline Pearcleaner/Resources/Localizable.xcstrings:4720]:
```suggestion
      }
    },
    "Locations that will be searched for .app files. Click a non-default path to remove it. Default paths can't be removed." : {
```

**Comment by alienator88** [comment]
I merged your changes but the strings catalog got corrupted so I had to revert.
Not sure what was done to the formatting of the strings catalog..

**Comment by kiwamizamurai** [comment]
@alienator88 

> I merged your changes but the strings catalog got corrupted so I had to revert.
Not sure what was done to the formatting of the strings catalog..

I'm not sure why it was happening. I made a new PR. it's working without any corruption

**Pull request merged**